# This patch file was generated by NetBeans IDE
# Following Index: paths are relative to: /space/src/hudson/main/core/src/main/resources/hudson
# This patch can be applied using context Tools: Patch action on respective folder.
# It uses platform neutral UTF-8 encoding and \n newlines.
# Above lines and this line are ignored by the patching process.
Index: Messages.properties
--- Messages.properties Base (BASE)
+++ Messages.properties Locally Modified (Based On LOCAL)
@@ -3,12 +3,12 @@
 FilePath.validateAntFileMask.doesntMatchAndSuggest=\
   ''{0}'' doesn''t match anything, but ''{1}'' does. Perhaps that''s what you mean?
 
-Util.second={0} {0,choice,0#seconds|1#second|1<seconds}
-Util.minute={0} {0,choice,0#minutes|1#minute|1<minutes}
-Util.hour  ={0} {0,choice,0#hours|1#hour|1<hours}
+Util.second={0} sec
+Util.minute={0} min
+Util.hour  ={0} hr
 Util.day   ={0} {0,choice,0#days|1#day|1<days}
-Util.month ={0} {0,choice,0#months|1#month|1<months}
-Util.year  ={0} {0,choice,0#years|1#year|1<years}
\ No newline at end of file
+Util.month ={0} mo
+Util.year  ={0} yr
 Util.countSuffix=s
 
 # ideally it should be "{0} ago" but this saves more space
